packages/web-components/src/components/rux-notification/rux-notification.scss

```diff
@@ -3,18 +3,17 @@
     --notification-banner-color-background: var(
         --color-background-base-default
     );
-    --notification-banner-border-width-left: 12px;
+    --notification-banner-border-width-left: var(--spacing-3);
     --notification-banner-radius-outer: var(--radius-base);
-    --notification-banner-radius-inner: 3px;
-    --notification-banner-border-width-large: 4px;
-    --notification-banner-padding: 0 0.5rem 0 1rem; //0 8px 0 16px
-    --notification-banner-border-width-small: 2px;
-    --notification-banner-inner-border-width: 1px;
+    --notification-banner-radius-inner: calc(var(--radius-base) / 3 * 2);
```

Comment:
just noting here: we'll eventually want this to scale w/ border width/outer radius. there's a separate RFC out for that at the token level.